x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/trusted_apps_page.test.tsx

```diff
@@ -30,7 +30,6 @@ import {
 } from '../../../../../../fleet/common';
 import { EndpointDocGenerator } from '../../../../../common/endpoint/generate_data';
 import { isFailedResourceState, isLoadedResourceState } from '../state';
-import { forceHTMLElementOffsetWidth } from './components/effected_policy_select/test_utils';
```

Comment:
Can you check if this utility is still being used (I don't think it is) and if not, just delete it.